blog/2021-03-16-BatchSolvingWithAmq.adoc

```diff
@@ -0,0 +1,182 @@
+= Batch solving in a stateless manner with OptaPlanner and AMQ
+:page-interpolate: true
+:imagesdir: 2021/03/16
+:awestruct-author: rsynek
+:awestruct-layout: blogPostBase
+:awestruct-tags: [amq, quarkus, messaging, integration]
+:awestruct-share_image_filename: architecture.png
```

Comment:
These blog images often end up in slides etc. Can we make it a bit more identifiable by renaming it to amqSolvingArchitecture.png?